knowledge-base/actions/mnajdova/github-action-required-labels/action-security.yml

```diff
@@ -0,0 +1,2 @@
+name: 'Require Labels'
+#Note: GITHUB_TOKEN not used
```

Comment:
I think there is `issues: read` permission, `to list labels on a pr`. view [here](https://github.com/mnajdova/github-action-required-labels/blob/a9a8832111eee32b8ed1128950f8896ad3346584/index.js#L27).